test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -359,7 +359,7 @@ Describe "Validate Update-Help from the Web for one PowerShell module for user s
         $ProgressPreference = $SavedProgressPreference
     }
 
-    RunUpdateHelpTests -Tag "CI" -Scope 'CurrentUser'
+    RunUpdateHelpTests -Tag:CI -UserScope
```

Comment:
Please use the common pattern here and below:
```suggestion
    RunUpdateHelpTests -Tag "CI" -UserScope
```

Reply:
As requested, although I have never seen a common pattern of using spurious double quotes in scripts.

Reply:
The "CI" string is often used in our tests, It is more easy discover one pattern if we will need to change this.